QtlTodoList/QtlTodoListApp.swift

```diff
@@ -6,12 +6,24 @@
 //
 
 import SwiftUI
+import FirebaseCore
 
+class AppDelegate: NSObject, UIApplicationDelegate {
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey : Any]? = nil) -> Bool {
+    FirebaseApp.configure()
+
+    return true
+  }
+}
 @main
 struct QtlTodoListApp: App {
+    // MARK: - Property Wrapper
+    @UIApplicationDelegateAdaptor(AppDelegate.self) var delegate
```

Comment:
```suggestion
     // MARK: - init
    init() {
        FirebaseApp.configure()
    }
```

[nits] こんな感じでinitでFirebaseのconfigureを呼ぶのもあり
-> AppDelegateが不要になる

ref. https://github.com/naoki-mrmt/VolleyballScoreApp/blob/develop/VolleyballScoreApp/Application/VolleyballScoreApp.swift

Reply:
initに変更しました！
a0220fc525adcbd7cdf8b4a18dc337a6c21f30ff